infra/scripts/migrate.sh

```
@@ -142,23 +144,61 @@ migrate_down() {
+    local version=$1
+    if [ -z "$version" ]; then
+        print_error "No version specified. Usage: make migrate-force version=18"
+        exit 1
+    fi
+    print_status "Forcing migration version to $version..."
+    if docker compose -f ./docker/docker-compose.yml ps mysql | grep -q "Up"; then
+        print_status "Docker MySQL container is running, forcing version inside Docker..."
+        check_go
+        wait_for_db
+        load_env
+        docker compose -f ./docker/docker-compose.yml exec api go run cmd/migrate/main.go -action=force -version=$version
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)